### DISCUSSION TITLE
Ideas for how to migrate staging data to production

### DISCUSSION THREAD

**Discussion by spencerflagg** [beginner-corp/begin-community#66] [Q&A] [answered]
Before I spend time rolling my own solution to move data from staging to production...

- is there a baked-in Begin process for doing this that i missed?
- has anyone built any generic tools for doing this?
- does the community have any suggestions or lessons learned for doing this?

Thanks much in advance!

**Comment by brianleroux** [comment, marked as answer]
Great idea! We do not have an official tool for this but we should. (Adding to our backlog.) I'll update here if we can cook up something in the short term.

**Reply by brianleroux** [reply]
Oh! And I forgot to mention we do have a download link available on the apps setting page look for "export your app data here" under the eject section.
